theories/_CoqProject

```diff
@@ -331,20 +331,19 @@ TM/Univ/StepTM.v
 TM/PrettyBounds/PrettyBounds.v
 TM/PrettyBounds/SizeBounds.v
 TM/PrettyBounds/MaxList.v
-# TM/PrettyBounds/BaseCode.v
-# TM/PrettyBounds/UnivBounds.v
-# TM/PrettyBounds/M2MBounds.v
+TM/PrettyBounds/BaseCode.v
+TM/PrettyBounds/UnivBounds.v
+TM/PrettyBounds/M2MBounds.v
 
-# TM/PrettyBounds/SpaceBounds.v
-# TM/PrettyBounds/BaseCodeSpace.v
-# TM/PrettyBounds/UnivSpaceBounds.v
+TM/PrettyBounds/SpaceBounds.v
```

Comment:
Are these files really needed for compilation? On the long run we shouldn't have such proofs in the undecidability library. Even on short term it might make sense to already now move them to the complexity library.

Reply:
Yes, we should move them out.